.travis.yml

```diff
@@ -7,6 +7,10 @@ branches:
 jobs:
   fast_finish: true
   include:
+    - php: 7.1.
+      env: DB=mysql COMPOSER_ARGS=--prefer-lowest
+      addons:
+        mariadb: '10.2'
```

Comment:
maybe pin mariadb to 10.2.2 (https://docs.neos.io/cms/installation-development-setup/system-requirements#db-compatibility-notes)?